packages/components/src/editable-page-header/index.js

```diff
@@ -13,17 +13,28 @@ import Form from './form';
  * Style dependencies
  */
 import './style.scss';
+import classnames from 'classnames';
 
-const EditablePageHeader = ( { onChange, text } ) => {
+const EditablePageHeader = ( { onChange, text, isEditDisabled } ) => {
```

Comment:
Nit: consider renaming the property to just `disabled` - that's inline with all our other inputs/buttons/form components. I'm a fan of keeping things as consistent as possible.

Reply:
I agree. Thanks for the suggestion. ✅